projects/core/components/notification/notification.style.less

```diff
@@ -81,5 +81,5 @@
     flex: 1;
     word-wrap: break-word;
     color: var(--tui-text-01);
-    hyphens: auto;
+    overflow: hidden;
```

Comment:
We do not need it here, otherwise it chops off extended parts of the letters on the last line. Why do you think we need it? Seems to work just fine without it.